admin/app/controllers/spree/admin/reimbursements_controller.rb

```diff
@@ -0,0 +1,50 @@
+module Spree
+  module Admin
+    class ReimbursementsController < ResourceController
+      belongs_to 'spree/order', find_by: :number
+
+      before_action :load_simulated_refunds, only: :edit
+      before_action :assign_performer, only: :perform
```

Comment:
@Vegann let's not do that in controller please, let's modify `Reimbursement#perform!`, change it to `def perform!(performer = nil)` and persist that in model